react/package.json

```diff
@@ -60,6 +62,8 @@
     "styled-components": "^6.1.12"
   },
   "peerDependencies": {
+    "@stripe/stripe-js": ">4",
```

Comment:
What happens if a schematic-react user doesn't have these peer dependencies installed?

Reply:
It depends on the package manager being used, but a missing peer dependency is typically automatically installed.